src/codec_aom.c

```diff
@@ -876,6 +876,11 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
                 return AVIF_RESULT_UNKNOWN_ERROR;
             }
         }
+
+        if (image->depth == 12) {
+            // The encoder may produce integer overflows with 12-bit input when loop restoration is enabled.
+            aom_codec_control(&codec->internal->encoder, AV1E_SET_ENABLE_RESTORATION, 0);
```

Comment:
Nit: Please check the return value. See line 875 for an example.

Reply:
I agree. I wanted to put this up first since I copied from another area in this function. Overall it's a bit inconsistent.

Reply:
Done. See line 859 for an example of an unchecked return.